cloud-consumption/src/main/java/com/sequenceiq/consumption/flow/consumption/storage/event/StorageConsumptionCollectionEvent.java

```diff
@@ -1,35 +1,27 @@
 package com.sequenceiq.consumption.flow.consumption.storage.event;
 
 import com.sequenceiq.cloudbreak.common.event.AcceptResult;
+import com.sequenceiq.consumption.flow.consumption.ConsumptionContext;
 import com.sequenceiq.flow.reactor.api.event.BaseFlowEvent;
 
 import reactor.rx.Promise;
 
 public class StorageConsumptionCollectionEvent extends BaseFlowEvent {
 
-    private final String environmentCrn;
+    private final ConsumptionContext context;
```

Comment:
I do not think it is a good idea to store the raw credentials in the database as part of the payload.